frontend/src/fixtures/leaderboardFixtures.js

```diff
@@ -7,7 +7,8 @@ const leaderboardFixtures = {
             "username": "one",
             "commonsId": 1,
             "totalWealth" : 1000,
-            "numOfCows": 5
+            "numOfCows": 5,
+            "numCowsSold": 3,
```

Comment:
@chriscerie made a comment on an earlier similar PR about newCowsBought that for new developers this variable could be confusing.  Changing it to `lifetimeCowsBought` and `lifetimeCowsSold` could make this more clear.

There's also an opportunity here to get more points than you would have otherwise.

I encourage you to mark both this PR and PR #17  as "draft PRs" and then submit new ones following the general pattern recommended here: 

https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685729994940309

You could 
* combine the fixture changes into a single PR for 10 points.
* changes to the UserCommons entity into a single PR for 10 points
* changes to the UserCommonsController into a single PR for 10 points
* changes to the ManageCows.js for 10 points
* changes to the leaderboard for 10 points.

That's 50 points instead of 40 points (20 each for this PR and #17 ... plus you will avoid merge conflicts in almost every file when you try to merge this (if you keep the two separate.)